crates/holochain/Cargo.toml

```diff
@@ -132,7 +132,7 @@ regex = "1.5"
 reqwest = "0.12"
 serial_test = "3.0"
 test-case = "3.3"
-tokio-tungstenite = "0.13"
+tokio-tungstenite = "0.21"
```

Comment:
Only used in tests, no changes required. I got errors upgrading `tunestenite` in the Holochain client but no problems here.